src/core/engines/engine_manager.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "core/engines/engine.hpp"
+#include "core/engines/engine_dispatcher.hpp"
+
+#include "engines/studio/input_selector/input_selector.hpp"
+#include "engines/studio/metronome/metronome.hpp"
+#include "engines/studio/mixer/mixer.hpp"
+#include "engines/studio/tapedeck/tapedeck.hpp"
+#include "core/audio/processor.hpp"
+
+namespace otto::engines {
+  using TapeTime = unsigned;
+
+  namespace tapeState {
```

Comment:
i think we should keep this interface for now, and then play around with it later. But my selection of avaliable functions here is kinda weird